.chloggen/unreleased/loki-resource-labels.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'bug_fix'
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: lokiexporter
```

Comment:
It should be `pkg/translator/loki` because changes are introduced there, not in lokiexporter component